demo/blocks_plug/run.py

```diff
@@ -1,36 +1,19 @@
 import gradio as gr
 
+import gradio as gr
```

Comment:
nit: We're re-importing gradio here and we're not using pandas

Reply:
Actually, should we move this to another demo? I think `blocks_plug` is a nice demo for how to set named tabs and don't want to lose it unless there's a good reason for it.

Reply:
Yeah something weird happened after a merge and I'm not sure what i did when resolving the conflict. I'll check this against main to see what is what and create a new demo for the tab changing if necessary.

Reply:
reinstated, no idea what happened.